scripts/firedrake-install

```diff
@@ -92,6 +92,8 @@ honoured.""",
                         help="C compiler to use when building with MPI (default is 'mpicc')")
     parser.add_argument("--show-petsc-configure-options", action="store_true",
                         help="Print out the configure options passed to PETSc")
+    parser.add_argument("--slepc", action="store_true",
+                        help="Install SLEPc along with PETSc")
```

Comment:
you also need to add slepc to the update case below. This is not just to make slepc work, at the moment args.slepc is not defined inside firedrake-update. This causes firedrake-update to fail every time at line 966!